packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -37,9 +37,113 @@ const handle = {
   handle(member) {
     const { node, parent, parentPath } = member;
 
+    if (member.isOptionalMemberExpression()) {
+      const root = member.find(({ node, parent, parentPath }) => {
+        if (parentPath.isOptionalMemberExpression()) {
+          return parent.optional || parent.object !== node;
```

Comment:
When is `parent.object !== node`? (Please, write a comment in the code)

Reply:
Done.